thrall/app/lib/ElasticSearch.scala

```diff
@@ -62,40 +63,56 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
       .executeAndLog(s"Indexing image $id")
       .incrementOnSuccess(indexedImages)}
 
-  def deleteImage(id: String)(implicit ex: ExecutionContext): Future[DeleteByQueryResponse] = {
-
-    val q = filteredQuery(
-      boolQuery.must(matchQuery("_id", id)),
-      andFilter(
-        missingOrEmptyFilter("exports"),
-        missingOrEmptyFilter(identifierField(Config.persistenceIdentifier)),
-        boolFilter.mustNot(termFilter(editsField("archived"), true)),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), "staff-photographer")),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), "contract-photographer")),
-        boolFilter.mustNot(termFilter(usageRightsField("category"), "commissioned-photographer"))
-      )
+  def deleteFilterQuery(id: String) = filteredQuery(
+    boolQuery.must(matchQuery("_id", id)),
+    andFilter(
+      missingOrEmptyFilter("exports"),
+      missingOrEmptyFilter(identifierField(Config.persistenceIdentifier)),
+      boolFilter.mustNot(termFilter(editsField("archived"), true)),
+      boolFilter.mustNot(termFilter(usageRightsField("category"), "staff-photographer")),
+      boolFilter.mustNot(termFilter(usageRightsField("category"), "contract-photographer")),
+      boolFilter.mustNot(termFilter(usageRightsField("category"), "commissioned-photographer"))
     )
+  )
 
-    val deleteQuery = client
+  private def deleteImageQuery(id: String) =
```

Comment:
It's a bit confusing that similarly named `deleteFilterQuery` and `deleteImageQuery` are two completely different things: one returns an ES query object, the other runs an ES  query command.